webclient/gulpfile.js

```diff
@@ -455,6 +456,27 @@ function copy_assets() {
 }
 gulp.task('assets', copy_assets);
 
+// --- Sitemap Pipeline ---
+function generate_sitemap() {
+    return gulp.src(['src/md/*.md', 'src/index.html'], { read: false })
+               .pipe(rename(path => {
+                   if (path.extname === '.md') {
+                       path.dirname = 'about';
+                       path.extname = '';
+                   } else {
+                       path.dirname = '';
+                   }
+               }))
+               .pipe(sitemap({
+                   siteUrl: 'https://nav.tum.sexy/',
+                   fileName: 'sitemap-webclient.xml',
+                   changefreq: 'weekly',
```

Comment:
More than the data, we dont change our about pages weekly. If we change something major, we can rerequest an indexing by google.
I would suggest `monthly`